doc/api/http.md

```diff
@@ -2639,6 +2639,7 @@ try {
 [`Buffer.byteLength()`]: buffer.html#buffer_class_method_buffer_bytelength_string_encoding
 [`Duplex`]: stream.html#stream_class_stream_duplex
 [`HPE_HEADER_OVERFLOW`]: errors.html#errors_hpe_header_overflow
+[`HPE_UNEXPECTED_CONTENT_LENGTH`]: errors.html#errors_hpe_unexpected_content_length
```

Comment:
This should only be added if there's a reference (e.g. ```[`HPE_UNEXPECTED_CONTENT_LENGTH`][]```) to it somewhere else in the `http.md` docs. If there's not, just remove this line.

Reply:
fixed